packages/bpk-component-scrollable-calendar/src/bpk-scrollable-calendar-grid.scss

```diff
@@ -0,0 +1,38 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '~bpk-mixins/index';
+
+.bpk-scrollable-calendar-grid {
+  width: 100%;
+  border-collapse: separate;
+  border-spacing: 0;
+
+  &--animating {
+    z-index: 1;
+    transition: transform .2s ease-in-out;
+  }
+
+  &__title {
+    margin: $bpk-spacing-md 0 $bpk-spacing-sm $bpk-spacing-sm;
+  }
+
+  &__month {
+    @include bpk-visually-hidden;
+  }
+}
```

Comment:
From what I can tell, `__animating` and `__month` aren't used, are they supposed to be in here?

Also, in the accompanying JS file a `__caption` class is used, but that isn't defined here.